### PR TITLE
Avoid UI obscuring code

### PR DESCRIPTION
**Pull request by jakearchibald**
Before:

jakearchibald github io_svgomg_ (1)

After:

localhost_8080_ (1)

### CODE REVIEW

**Review by TrySound — APPROVED** [review]

File: src/css/components/_code-output.scss [hunk 1]
```diff
@@ -25,6 +25,11 @@
     font-size: 0.9rem;
     display: block;
     padding: 14px;
+
+    @media (min-width: 640px) {
+      // Allow the code area to scroll past the floating action buttons.
+      margin-bottom: 200px;
```
Comment (XhmikosR) [inline src/css/components/_code-output.scss:31]: @jakearchibald I was thinking, maybe 200px is too much?

Comment (jakearchibald) [reply src/css/components/_code-output.scss:31]: Hm, it can't be much less before the code goes under the background-change icon.

Comment (jakearchibald) [reply src/css/components/_code-output.scss:31]: It looks like there's a lot of space in the screenshot above, but the text isn't all the way across on the final line.

Comment (XhmikosR) [reply src/css/components/_code-output.scss:31]: Yeah, that's what I meant. Not sure why this happens yet.

**Comment by XhmikosR** [comment]
@jakearchibald Another thing I notice after this PR is that when cache is empty, after clicking on Demo, the transition is a bit sluggish. It's like something is firing up at the same time; maybe the loader and something else?

**Comment by jakearchibald** [comment]
I'm pretty sure it's not this PR causing it. The changed style only resolves when you go to the code view, which isn't the default view.

**Pull request merged**